crates/nu-utils/standard_library/std.nu

```diff
@@ -296,3 +296,721 @@ export def "log debug" [message: string] {
 
     print --stderr $"(ansi default_dimmed)DEBUG ($message)(ansi reset)"
 }
+
+def error-fmt [] {
+    $"(ansi red)($in)(ansi reset)"
+}
+
+def throw-error [error: string, msg: string, span: record] {
+    error make {
+        msg: ($error | error-fmt)
+        label: {
+            text: $msg
+            start: $span.start
+            end: $span.end
+        }
+    }
+}
+
+def module-not-found-error [span: record] {
+    throw-error "std::help::module_not_found" "module not found" $span
+}
+
+def alias-not-found-error [span: record] {
+    throw-error "std::help::alias_not_found" "alias not found" $span
+}
+
+def extern-not-found-error [span: record] {
+    throw-error "std::help::extern_not_found" "extern not found" $span
+}
+
+def operator-not-found-error [span: record] {
+    throw-error "std::help::operator_not_found" "operator not found" $span
+}
+
+def command-not-found-error [span: record] {
+    throw-error "std::help::command_not_found" "command not found" $span
+}
+
+def print-help-header [
+    text: string
+    --no-newline (-n): bool
+] {
+    let header = $"(ansi green)($text)(ansi reset):"
+
+    if $no_newline {
+        print -n $header
+    } else {
+        print $header
+    }
+}
+
+def show-module [module: record] {
+    if not ($module.usage? | is-empty) {
+        print $module.usage
+        print ""
+    }
+
+    print-help-header -n "Module"
+    print $" ($module.name)"
+    print ""
+
+    if not ($module.commands? | is-empty) {
+        print-help-header "Exported commands"
+        print -n "    "
+
+        let commands_string = (
+            $module.commands
+            | each {|command|
+                $"($command) (char lparen)(ansi cyan_bold)($module.name) ($command)(ansi reset)(char rparen)"
+            }
+            | str join ", "
+        )
+
+        print $commands_string
+        print ""
+    }
+
+    if not ($module.aliases? | is-empty) {
+        print-help-header -n "Exported aliases:"
+        print $module.aliases
+        print ""
+    }
+
+    if ($module.env_block? | is-empty) {
+        print $"This module (ansi cyan)does not export(ansi reset) environment."
+    } else {
+        print $"This module (ansi cyan)exports(ansi reset) environment."
+        view source $module.env_block | nu-highlight
```

Comment:
This one currently shows as a huge red blob. Prepending `export-env` before the block should fix it. Although I'm thinking if the `export-env` block is too big, it might be quite annoying (see https://github.com/pypa/virtualenv/blob/main/src/virtualenv/activation/nushell/activate.nu, for example), we can just leave it out as well. You can use `view source module-name` to view the source code if you want.

Reply:
mm you are rigth
```bash
use crates/nu-utils/standard_library/std.nu 'help modules'
help modules std
```
gives a red blob :thinking:

Reply:
i removed it for now in 1e14b4620b5b3b4fa406c381e3a72e72581ffde5